File: src/main/java/com/github/clojure_lsp/intellij/Icons.java
package com.github.clojure_lsp.intellij;

import com.intellij.openapi.util.IconLoader;
import javax.swing.Icon;

public interface Icons {
  Icon ClojureFile = IconLoader.getIcon("/clojure-lsp/icons/clojure.svg", Icons.class);
  Icon StatusConnected = IconLoader.getIcon("/clojure-lsp/icons/clojure.svg", Icons.class);
  Icon StatusDisconnected = IconLoader.getIcon("/clojure-lsp/icons/clojure-mono.svg", Icons.class);
}

File: src/main/kotlin/icons.kt
package com.github.clojure_lsp.intellij

import com.intellij.openapi.util.IconLoader
import javax.swing.Icon

object Icons {
  @JvmField val CLOJURE = IconLoader.getIcon("/icons/clojure.svg", Icons::class.java)
  @JvmField val STATUS_CONNECTED = IconLoader.getIcon("/icons/clojure.svg", Icons::class.java)
  @JvmField val STATUS_DISCONNECTED = IconLoader.getIcon("/icons/clojure-mono.svg", Icons::class.java)
}
